File: Logic_HandsOn/C++/Problems on Digits,String,Array,Pointers etc/program25.cpp
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		a program which accept area in square feet and convert it into square meter. 
//		(1 square feet = 0.0929 Square meter)
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////



#include<iostream>
using namespace std;

class Square
{
	private:
		int iValue;
		double dSquareMeter;

	public:
		Square(int iValue,double SquareMeter=0.0)
		{
			this->iValue=iValue;
		}

		double SquareMeter()
		{
			dSquareMeter = iValue*0.0929;
			return dSquareMeter;
		}
};		

int main()
{
	int iValue = 0;
	double dRet = 0.0;

	cout<<"Enter area in square feet ";
	cin>>iValue;

	Square *sObj = new Square(iValue);
	dRet = (*sObj).SquareMeter();

	cout<<dRet<<"\n";

}

/*
	OUTPUT:
		
		Enter area in square feet 5
		0.464500

		Enter area in square feet 7
		0.650300
*/

File: Logic_HandsOn/Java/Problems on Array,Pattern etc/Program25.java
/////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		a program which accept area in square feet and convert it into square meter. 
//		(1 square feet = 0.0929 Square meter)
//
////////////////////////////////////////////////////////////////////////////////////////////////////////////////

import java.util.*;

class Solution
{
	public float iValue;

	public Solution(float iValue)
	{
		this.iValue = iValue;
	}

	public double SquareMeter()
	{

                double dSq = iValue*0.0929;      	
		return dSq;
	
	}
}

class Program
{
	public static void main(String[] args)
	{
		Scanner sObj = new Scanner(System.in);

		double dRet=0.0;

		System.out.println("Enter Radius: ");
		float fValue = sObj.nextFloat();

		Solution obj = new Solution(fValue);
		dRet = obj.SquareMeter();
	
		System.out.println("Conversion is: "+dRet);
	}
}

/*
OUTPUT:

Enter Radius: 
5
Conversion is: 0.46449999999999997

*/
